Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -5,11 +5,19 @@
 public class PlayerMovement : MonoBehaviour
 {
     [SerializeField]
-    private float moveSpeed;
+    private float moveSpeed = 4f;
 
     private Animator animator;
     private new Rigidbody2D rigidbody;
     private Vector3 positionChange;
+    private float grabFloat = 0;
+    private bool canGrab;
+    private GameObject handleObject;
+    private Transform handleTransform;
+    private string handleName;
+    private Vector2 handleOrientation;
```

Comment:
It would probably make more sense to store information about handles in their own Handle class, that way all you need is the handle GameObject and you can get all of the info you need from the handle itself by calling properties off of the script when you need to use them.